controllers/tests/controller_server/common.py

```diff
@@ -17,6 +17,15 @@ def mock_array_type(contex, server_path):
     contex.addCleanup(detect_array_type_patcher.stop)
 
 
+def mock_utils_method(contex, utils_method_name, return_value=None):
+    utils_method_path = '.'.join(('controllers.servers.utils', utils_method_name))
```

Comment:
can you move `controllers.servers.utils` to test_settings?

Reply:
settings is mainly for consts..

Reply:
I think it's also a const, but ok, it doesn't necessary